src/filetest.c

```diff
@@ -0,0 +1,71 @@
+#include "filetest.h"
```

Comment:
Why is this called `filetest.c`?

Reply:
This is no longer used, so we may remove it.